frontend/src/worksheets/FilterWorkSheets.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css'
+
+function FilterWorkSheets(props) {
+  const { status, onStatusChange } = props
+
+  const handleInputChange = (event) => {
+    onStatusChange(event.target.value)
+  }
+
+  return (
+    <div>
+      <form>
+        <select
+          value={status}
```

Comment:
Van default értéke ennek a selectnek vagyis a statusnak?
Ha nincs, akkor akkor előfordulhat, hogy filterezéskor a `getWorkSheets() undef vagy üres stringet kap statusként... 

Ez kezelve lesz valahogy?

Reply:
Köszönöm a jelzést. Most már van defaultja.

Reply:
Illetve volt defaultja. Az üres String ami mindet visszaadja. Arra állítom vissza és tesztelem.

Reply:
Írtam egyet a CREATED default beállításához, azt kellene megbeszélni és utána az alapján tovább menni.

Reply:
A kérés akkor az, hogy alapesetben ne legyen szűrés. Vagyis mindet betöltse. Ehhez a default értéket ""-re állítottam. Az api erre minden listázza.